legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-snowflake/src/main/java/org/finos/legend/engine/persistence/components/relational/snowflake/sql/visitor/CopyVisitor.java

```diff
@@ -89,7 +94,17 @@ else if (format instanceof StandardFileFormat)
                 }
                 copyStatement.setFileFormatType(standardFileFormat.formatType());
                 copyStatement.setFileFormatOptions(formatOptions);
+
+                if (standardFileFormat.formatType().equals(AVRO))
+                {
+                    selectFields = current.avroSourceDataset().get();
+                }
+                else
+                {
+                    selectFields = current.sourceDataset();
```

Comment:
Remove the else block and have this as the default value for that variable. Or the value if selectFields is null.